.github/pull_request_template.md

```diff
@@ -0,0 +1,34 @@
+## Description
+A few sentences describing the overall goals of the pull request's commits.
+
+## Feature Status
+**READY/IN DEVELOPMENT**
+
```

Comment:
Maybe add a section to list all the issues that this pull request fixes, if any, e.g.
```text
## Issues This PR Fixes
Fixes #NNNN 
Fixes #NNNN 
```
GitHub is smart: once this PR gets merged, it will automatically close those issues!